pywhat/Data/regex.json

```diff
@@ -238,7 +225,7 @@
       "Tags": [
          "Identifiers",
          "Networking",
-         "IPv6"
```

Comment:
The reason we used specific tags here was so people could pick and choose what they wanted specifically, if we can filter on names this isn't needed :)

Reply:
IPv4 and IPv6 are the ones that may actually come useful, so they can be added back yeah. Are there any others?

Reply:
In this case, it may be better to add individual tags to all entries.